tests/verify_changes.py

```
@@ -0,0 +1,134 @@
+            print(f"❌ Unexpected return length: {len(res)}. Expected 4.")
+            
+    except Exception as e:
+        print(f"❌ Crash during backward compatibility test: {e}")
+        import traceback
+        traceback.print_exc()
+
+def test_logistic_interactions(df):
+    print("\n[TEST 2] Logistic Regression with Interactions")
+    try:
+        pairs = [('age', 'sex'), ('treatment', 'age')]
+        html, or_res, aor_res, int_res = analyze_outcome(
```

Ruff (0.14.10)

Unpacked variable `html` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)